composer.json

```diff
@@ -51,9 +51,11 @@
         "mezzio/mezzio-fastroute": "^3.0",
         "mezzio/mezzio-laminasrouter": "^3.0",
         "mockery/mockery": "^1.0",
-        "phpstan/phpstan": "^0.9.2",
-        "phpstan/phpstan-strict-rules": "^0.9",
-        "phpunit/phpunit": "^7.0.1"
+        "phpstan/extension-installer": "^1.1",
+        "phpstan/phpstan": "^0.12",
+        "phpstan/phpstan-phpunit": "^0.12",
+        "phpstan/phpstan-strict-rules": "^0.12",
+        "phpunit/phpunit": "^9.3"
```

Comment:
Hm, I thought, the TSC decided to use psalm, and not phpstan?

Reply:
Yeah, but the tooling was pre-installed/running, so it makes sense to upgrade them for now, and do the psalm migration separately.